core.rb

```diff
@@ -663,6 +674,23 @@ def Core.handle_repo_does_not_exist(repos_url, issue_number, login, close=true)
     return "repos does not exist"
   end
 
+  def Core.handle_caps_check_failed(repos_url, issue_number, login, close=true)
+    comment = <<-END.unindent
+      Dear @#{login} ,
+
+      The github link provided above https://github.com/#{repos_url} ,
+      does not match the capitalization of the github repository.
+      Please update.
+    END
+    if close
+      comment += "I am closing this issue. Please try again with a new issue."
+      Core.close_issue(issue_number)
+    end
+    Octokit.add_comment(Core::NEW_ISSUE_REPO, issue_number, comment)
```

Comment:
Should the order of close / add_comment be reversed here, so that the comment is added and then the issue closed?

Reply:
I modelled it after the handle_repo_does_not_exist  code that was already present but yes the order should probably change.